"next": "8.0.0-canary.16" cannot find module '../server/htmlescape'

Update from 14 to 16 (with Acorn workarouch, which works for 14)

{ Error: Cannot find module '../server/htmlescape'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.../server/htmlescape (/Users/steida/dev/este/.next/server/static/development/pages/_document.js:103:18)
    at __webpack_require__ (/Users/steida/dev/este/.next/server/static/development/pages/_document.js:23:31)
    at Object../node_modules/next/dist/pages/_document.js (/Users/steida/dev/este/.next/server/static/development/pages/_document.js:1629:20)
    at __webpack_require__ (/Users/steida/dev/este/.next/server/static/development/pages/_document.js:23:31)
    at Object../node_modules/next/document.js (/Users/steida/dev/este/.next/server/static/development/pages/_document.js:2013:18)
    at __webpack_require__ (/Users/steida/dev/este/.next/server/static/development/pages/_document.js:23:31)
    at Module../pages/_document.tsx (/Users/steida/dev/este/.next/server/static/development/pages/_document.js:2835:71)
    at __webpack_require__ (/Users/steida/dev/este/.next/server/static/development/pages/_document.js:23:31)
    at Object.1 (/Users/steida/dev/este/.next/server/static/development/pages/_document.js:2992:18)
    at __webpack_require__ (/Users/steida/dev/este/.next/server/static/development/pages/_document.js:23:31)
    at /Users/steida/dev/este/.next/server/static/development/pages/_document.js:91:18
    at Object.<anonymous> (/Users/steida/dev/este/.next/server/static/development/pages/_document.js:94:10) code: 'MODULE_NOT_FOUND' }

--- a/packages/next/build/webpack-config.js
+++ b/packages/next/build/webpack-config.js
@@ -43,7 +43,7 @@ function externalsConfig (isServer, target) {
       }
 
       // Default pages have to be transpiled
-      if (res.match(/next[/\\]dist[/\\]pages/) || res.match(/next[/\\]dist[/\\]client/) || res.match(/node_modules[/\\]@babel[/\\]runtime[/\\]/) || res.match(/node_modules[/\\]@babel[/\\]runtime-corejs2[/\\]/)) {
+      if (res.match(/next[/\\]dist[/\\]/) || res.match(/node_modules[/\\]@babel[/\\]runtime[/\\]/) || res.match(/node_modules[/\\]@babel[/\\]runtime-corejs2[/\\]/)) {
         return callback()
       }
 

